recOrder/plugin/widget/main_widget.py

```diff
@@ -1799,21 +1799,25 @@ def handle_fluor_reconstructor_update(self, value):
 
     @pyqtSlot(dict)
     def handle_meta_update(self, meta):
-        if self.last_calib_meta_file is None:
-            print("\nWARNING: No calibration file has been loaded\n")
-            return
+        # Don't update microscope parameters saved in calibration metadata file
 
-        with open(self.last_calib_meta_file, 'r') as file:
-            current_json = json.load(file)
-
-        for key, value in current_json['Microscope Parameters'].items():
-            if key in meta:
-                current_json['Microscope Parameters'][key] = meta[key]
-            else:
-                current_json['Microscope Parameters'][key] = None
+        # if self.last_calib_meta_file is None:
+        #     print("\nWARNING: No calibration file has been loaded\n")
+        #     return
+        #
+        # with open(self.last_calib_meta_file, 'r') as file:
+        #     current_json = json.load(file)
+        #
+        # for key, value in current_json['Microscope Parameters'].items():
+        #     if key in meta:
+        #         current_json['Microscope Parameters'][key] = meta[key]
+        #     else:
+        #         current_json['Microscope Parameters'][key] = None
+        #
+        # with open(self.last_calib_meta_file, 'w') as file:
+        #     json.dump(current_json, file, indent=1)
 
-        with open(self.last_calib_meta_file, 'w') as file:
-            json.dump(current_json, file, indent=1)
+        pass
```

Comment:
I've realized that the acquisition works try to do a weird update of the calibration metadata. I think the intent is to update the microscope parameters (such as NA or magnification) in the metadata file. However, we currently (at least in most of my use cases) don't enter such microscope parameters during LC calibration, and they are also not written at that time - `write_metadata` is typically called without the optional `microscope_params` argument. So this function will have no effect, and in fact it throws an error since we changed the `Microscope Parameters` key to `Microscope parameters`. 

The only time we currently write these microscope parameters is during background capture: https://github.com/mehta-lab/recOrder/blob/1318f2d50d9cb9057822bdb2e6016ec79ac05f3a/recOrder/plugin/workers/calibration_workers.py#L321

I think it's good to save some of these parameters (such as magnification, n_objective_media, objective_NA, condenser_NA, swing, pixel_size) in the calibration metadata. Parameters such as pad_z or bg_correction_option belong to the dataset metadata, not the calibration metadata.

I welcome help and input on this issue

Reply:
Great, thanks for the context here. I'll think more about what parameters should be saved alongside the calibration metadata. I'll probably break it off into a separate issue/PR since there's a bit of work to do here.